Rocket.Chat/Controllers/Subscriptions/NewRoomViewController.swift

```diff
@@ -182,8 +182,8 @@ class NewRoomViewController: BaseViewController {
                 SubscriptionManager.updateSubscriptions(auth) {
                     if let newRoom = Realm.current?.objects(Subscription.self).filter("name == '\(name)' && privateType != 'd'").first {
 
-                        let controller = ChatViewController.shared
-                        controller?.subscription = newRoom
+                    // let controller = ChatViewController.shared
+                    // controller?.subscription = newRoom
```

Comment:
Regression: not entering channel after creating it

Reply:
✅